frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/treat-patients/patient-details/simulated-region-overview-behavior-treat-patients-patient-details.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="col-sm d-inline-flex justify-content-center">
+    <app-patient-status-badge
+        [status]="(visibleStatus$ | async)!"
+    ></app-patient-status-badge>
+</div>
+<div class="col-sm d-inline-flex justify-content-center">
```

Comment:
Maybe also display a column for the real status

Reply:
Christian said this wasn't needed, only the Verlauf